librecad/src/ui/forms/qg_commandwidget.cpp

```diff
@@ -155,7 +157,48 @@ void QG_CommandWidget::tabPressed() {
             leCommand->setText(reducedChoice.first());
         }
         else if (reducedChoice.count()>0) {
+            QString longestString = "";
+            QString shortestString = "";
```

Comment:
Yep, I know, but I keep this habit for all data types, even for those which does not need it.

I pushed some commits to fix the behavior I had with the bisection, and created a slot for this method.
I didn't know that pushing on a branch after a PR added new changes to that PR. It's useful to know this.